File: 01.CSharp-Basics/13.NestedLoopsLab/Combinations/Combinations.cs
namespace Combinations
{
    using System;
    public class Combinations
    {
        public static void Main(string[] args)
        {
            int counter = 0;
            int number = int.Parse(Console.ReadLine());
            for (int i = 0; i <= number; i++)
            {
                for (int j = 0; j <= number; j++)
                {
                    for (int k = 0; k <= number; k++)
                    {
                        if (i + j + k == number)
                        {
                            counter++;
                        }
                    }
                }
            }

            Console.WriteLine(counter);
        }
    }
}

File: JavaScript/01.JavaScriptBasic/11Nested-Loops-Lab/src/combinations.js
function calcDesicion(params) {
    let number = Number(params[0]);
    let counter = 0;
    for (let a = 0; a <= number; a++) {
        for (let b = 0; b <= number; b++) {
            for (let c = 0; c <= number; c++) {
                if (a + b + c === number) {
                    counter++;
                }
            }
        }
    }

    console.log(counter);
}

calcDesicion(["25"]);